app/services/cosmos/cosmos.ts

```diff
@@ -84,10 +84,14 @@ export class CosmosAPI {
   /**
    * Get the list of validators
    */
-  async getValidators(): Promise<CosmosValidator[]> {
+  async getValidators({ paginationKey = undefined } = {}): Promise<CosmosValidator[]> {
     const bondStatus = '' as BondStatusString
-    const { validators } = await this.queryClient.staking.validators(bondStatus)
-    return validators.map(v => convertValidator(v))
+    const { validators, pagination } = await this.queryClient.staking.validators(bondStatus, paginationKey)
```

Comment:
might have off by one issue since https://github.com/cosmos/cosmos-sdk/pull/11408
need to mitigate with `0x0` next_key for first query